packages/api/tests/lib/test-publishSnsMessageUtils.js

```diff
@@ -164,3 +166,54 @@ test.serial('publishCollectionSnsMessage() publishes an SNS message for the even
   t.is(message.event, 'Delete');
   t.is(message.deletedAt, deletedAt);
 });
+
+test.serial('publishPdrSnsMessage() does not publish an SNS message if pdr_sns_topic_arn is undefined', async (t) => {
+  const newPdr = fakePdrFactoryV2({
+    pdrName: 'test_pdr',
+  });
+  await t.throwsAsync(
+    publishPdrSnsMessage(newPdr),
+    { message: 'The pdr_sns_topic_arn environment variable must be set' }
+  );
```

Comment:
So if that call throws an error, that means an error is thrown in `writePdr` as well if one is not defined. Is the upstream error handling robust enough to handle that error begin thrown? I know we don't rollback, which is good, but it seems we log a fatal error.. Is that the result we are looking for? It's okay if you aren't sure.

Reply:
Yeah, I believe it does handle the error being thrown in writePdr and I believe that is the result we are looking for 👀  I could be mistaken 

[`5c5141b` (#2449)](https://github.com/nasa/cumulus/pull/2449/commits/5c5141b8add72894f1459a4afb67439c7273ba09)